### PR TITLE
Update foundation-delegation-program.md

### PR DESCRIPTION
**Pull request by mindstyle85**
## Summary by CodeRabbit

- **New Features**
	- Updated compliance requirements for the Foundation Delegation Program to exclude infrastructure operations in Hetzner or OVH, enhancing selection criteria.
  
- **Documentation**
	- Revised the Foundation Delegation Program documentation to reflect the new infrastructure location requirements.

### CODE REVIEW

**Comment by mindstyle85** [comment]
@jcstein can you merge this for me pls?

**Review by jcstein — APPROVED** [review]

File: community/foundation-delegation-program.md [hunk 1]
```diff
@@ -83,6 +83,7 @@ compliance screen
 * Dedicated email address so that the Foundation can reach you in the event
 of emergency upgrades and fixes
 * Maintain a fully archival (non pruned) bridge node for both Mainnet Beta and Mocha if selected for the program
+* Not running your infra in Hetzner or OVH
```
Comment (jcstein) [inline community/foundation-delegation-program.md:86]: should this acronym be clarified @mindstyle85 ?

Comment (mindstyle85) [reply community/foundation-delegation-program.md:86]: you mean "infra"?

Comment (mindstyle85) [reply community/foundation-delegation-program.md]: fixed it :)

Comment (jcstein) [reply community/foundation-delegation-program.md]: i meant OVH but all g

**Pull request merged**